frontend/cypress.example.json

```diff
@@ -0,0 +1,7 @@
+{
+  "baseUrl": "http://kitspace.test:3000",
+  "env": {
+    "gitea_admin_username": "your gitea admin username",
+    "gitea_admin_password": "your gitea admin password"
+  }
+}
```

Comment:
Can we grab this from the .env ?

Reply:
No, it isn't OS env https://docs.cypress.io/guides/guides/environment-variables.html#Setting

Reply:
Oh, I think that is possible https://docs.cypress.io/guides/guides/environment-variables.html#Option-3-CYPRESS

Reply:
Done!